docs/governance/governance.md

```diff
@@ -230,6 +230,19 @@ To amend the Charter, Governance or [Project Policies](policies.md#project-polic
 
 The Steering Committee will meet as needed, but at least once per quarter. The Chair is responsible for organizing the meeting. Any Member can request the Chair to organize a meeting. Meetings are announced at least 2 weeks in advance, unless _all_ Members have agreed that a meeting on shorter notice is required.
 
+#### 6.4.1 Meeting minutes
+
+For every meeting of the Steering Committee, the Chair ensures that meeting minutes are taken and stored in a private location.
+The minutes will be made publicly available no later than four weeks after each meeting, and will provide a clear overview of discussed items, decisions made, and action points.
+Prior to publication, information that is considered to be confidential, sensitive, or for another reason not appropriate for public disclosure will be removed or redacted.
```

Comment:
We should also mention where the SC meeting notes will be publicly available (https://eessi.io/docs/governance/steering-committee/meeting-notes would make sense)

Reply:
Added: https://github.com/EESSI/docs/pull/628/commits/3666ede4a413bc6ff215464820f4c4300d556d90.